code/modules/admin/IsBanned.dm

```diff
@@ -97,10 +97,9 @@
 			qdel(exist_query)
 		else
 			if(!exist_query.NextRow()) // If there isnt a row, they aint been seen before
-				if(GLOB.panic_bunker_enabled)
+				if(SSqueue && SSqueue.queue_enabled && (length(GLOB.clients) > SSqueue.queue_threshold) && !(ckey in SSqueue.queue_bypass_list)) // To the person who tells me in code review "Oh you should use ?. here". No. That logic isnt appropriate here. It needs a dedicated null check due to the length comparison.
```

Comment:
`?.` would work because if it failed, the rest would simply not be evaluated.

Reply:
I'm curious what affected meant by this